datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -1147,6 +1147,7 @@ fn retrieve_rel_common(rel: &Rel) -> Option<&RelCommon> {
             RelType::Window(w) => w.common.as_ref(),
             RelType::Exchange(e) => e.common.as_ref(),
             RelType::Expand(e) => e.common.as_ref(),
+            RelType::Update(_) => None,
```

Comment:
The newly added [UpdateRel](https://github.com/substrait-io/substrait/blob/3d2ff77575a7177f82a4d5b53408a059e9818922/proto/substrait/algebra.proto#L669) does not have a RelCommon.